lib/binding_web/test/node.test.ts

```diff
@@ -0,0 +1,591 @@
+import { describe, it, expect, beforeAll, beforeEach, afterEach } from 'vitest';
+import type { Parser as ParserType, Language, Tree, Node } from '../src';
```

Comment:
Why can't `Parser` just be imported from `../src`, as opposed to returning it from the `helper`? I would think that the helper would just provide the languages.

Reply:
We can't, because we have to ensure that the module's been initialized. Importing `Parser` directly and using that here doesn't guarantee that, which is why we use the one exported from the helper. We could ensure it's initialized in all the tests, though I'm not sure if that's necessary.

Reply:
I agree that we need to await the promise returned by the helper, but I don't think there's any advantage of importing the `Parser` class itself from `helper`. You can just import `Parser` from `src`  at the top.

As written, this test makes it look like `web-tree-sitter` has a super weird API where you have to import the Parser type separately from the Parser class itself, and do a manual cast to `ParserType`.

Reply:
Yeah you're right, I had sworn I'd tried that and it had errored out for me, but it seems to work now.